src/Mod/BIM/nativeifc/ifc_objects.py

```diff
@@ -116,6 +117,35 @@ def onDocumentRestored(self, obj):
             QtCore.QTimer.singleShot(100, obj.Document.recompute)
             QtCore.QTimer.singleShot(100, self.fit_all)
 
+    def Classification(self, obj):
```

Comment:
Function names should begin with underscore (and use snake_case), that's a norm in Python, and we're trying to slowly adopt it in FreeCAD (although it's far from being done :sweat_smile: ). Class names can start with a capital letter.

```suggestion
    def assign_classification(self, obj):
```

Reply:
Done.